config/initializers/omniauth.rb

```diff
@@ -1,3 +1,11 @@
+if Settings.developer_auth
+  Rails.application.config.middleware.use OmniAuth::Builder do
```

Comment:
test coverage?

Reply:
Yea i didn't load in the test env so it's not available. This is done once at boot time. If it does get loaded it changes the priority of the other auths so wasn't how to best test this